tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -133,6 +147,16 @@ describe('Discover', function() {
       wrapper.update();
     });
 
+    it('resets query builder and state', function() {
+      wrapper.instance().reset();
```

Comment:
I think there's some value in testing the contract of the the Reset button itself with the reset handler.

Reply:
Good point, so far have just tested functions in discover, not any jsx etc but will keep this in mind